study.md

```diff
@@ -31,7 +33,7 @@ In your own words explain "this".  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+It is a shorthand placeholder for referncing an object within a method.
```

Comment:
Close, the method is actually within the object. When we use `this` in a method it refers to the object that stores the method.